src/components/tooltip/tooltip.js

```diff
@@ -442,9 +479,9 @@ function MdTooltipRegistry() {
 
   /**
    * Removes an event handler from the service.
-   * @param  {String} type Type of event handler.
-   * @param  {Function} handler The event handler itself.
-   * @param  {Boolean} useCapture Whether the event handler used event capturing.
+   * @param {string} type Type of event handler.
+   * @param {function} handler The event handler itself.
+   * @param {boolean} useCapture Whether the event handler used event capturing.
```

Comment:
@ErinCoughlan The functions that are being sent in are event handlers. There doesn't seem to be any params or return values, except for the `debouncedOnResize` function which is `$$rAF.throttle(updatePosition)`. For these cases, what would I need to add to make it more readable?